ironmon_tracker/Program.lua

```diff
@@ -1165,6 +1188,16 @@ function Program.isInStartMenu()
 	return startMenuWindowId == 1
 end
 
+---Forcibly change the in-game option for "Button Mode" from "HELP" to "LR"; allowing additional Tracker controls
+function Program.changeGameSettingForLR()
+	local addr2 = Utils.getSaveBlock2Addr()
+	local currentSetting = Memory.readbyte(addr2 + Program.Addresses.offsetOptionsButtonMode)
+	-- 0:NORMAL(HELP), 1:LR, 2:L_EQUALS_A
+	if currentSetting == 0 then
```

Comment:
Actually, should this *also* check for if button mode is L=A? Since the setting is worded as "Override Button Mode to LR" it'd make sense to me to also change from L=A

Functionally doesn't change a huge amount, but if it's on L=A then there's a potential annoyance if the player has both the route/trainer info open and an in-game menu open and wants to use L to cycle to the previous page

Reply:
I dont think so. If people intentionally change their setting cause they like using L as A, then i want to leave it like that. The important thing is R being the access button for the other helpful stuffs.

We'll find out in time if people run into an issue with this, and if so we can change it.